ci-operator/config/openshift/kuryr-kubernetes/openshift-kuryr-kubernetes-master.yaml

```diff
@@ -0,0 +1,52 @@
+tag_specification:
+  name: '4.0'
```

Comment:
Don't we now want `master` files to target `4.1`? @stevekuznetsov ?

Reply:
we do

Reply:
Fixed, thanks!